js/messaging.js

```diff
@@ -233,12 +237,42 @@ function openURL({
       url.replace(/%2F.*/, '*').replace(/#.*/, '') :
       url.replace(/#.*/, '');
 
-  return queryTabs({url: urlQuery, currentWindow}).then(maybeSwitch);
+  return manageMatch || editMatch ? queryTabs().then(maybeSwitch) :
+  queryTabs({url: urlQuery, currentWindow}).then(maybeSwitch);
 
   function maybeSwitch(tabs = []) {
     const urlWithSlash = url + '/';
     const urlFF = FIREFOX && url.replace(/%2F/g, '/');
-    const tab = tabs.find(({url: u}) => u === url || u === urlFF || u === urlWithSlash);
+    const urlOptions = manageMatch ? URLS.ownOrigin + 'manage.html?#stylus-options' : null;
+    const urlManage = manageMatch ? URLS.ownOrigin + 'manage.html' : null;
+    let tab = tabs.find(({url: u}) => u === url || u === urlFF || u === urlWithSlash || u === urlOptions || u === urlManage);
+    if (!tab && prefs.get('openEditInWindow') && chrome.windows && editMatch) {
```

Comment:
eslint: `prefs` should be listed in `globals` comment at the top

Reply:
Finally got around to fixing all these syntax issues, except the pref checks. They work, and that's how we do it everywhere. IDK what you're suggesting I put where.

Also, now that eslint is finally loading, I see an unrelated error for `CHROME_HAS_BORDER_BUG` being unused.

Reply:
I meant the comment at the top of the file that lists all global things. If you add `prefs` there, it'll pacify the linter.

Reply:
That's what I thought, but damned if I can get it to actually pacify the linter. It works for you? Does the order matter, or something?

Reply:
My bad, I didn't see the first word was `exported` so I mislead you.
Add `/* global prefs */` on a new line after the first comment.

Reply:
Thanks, will do.